core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -341,8 +352,8 @@ class ByteEfficiencyAudit extends Audit {
         }
       }
 
-      metricSavings.FCP = fcpSavings;
-      metricSavings.LCP = Math.max(lcpGraphSavings, lcpRecordSavings);
+      metricSavings.FCP = fcpOptimisticSavings;
```

Comment:
nit: Adding "optimistic" to the variable representing savings could imply that this is a high-ball estimate. I think `fcpSavings` is preferable here.